src/app/styles/js/shared.js

```diff
@@ -3,33 +3,39 @@ import IconButton from '@material-ui/core/IconButton';
 import Card from '@material-ui/core/Card';
 
 // Styles for overriding material UI
-// General colors
-//
-export const white = '#fff';
-export const black = '#000';
-export const alertRed = '#d0021b';
-export const checkBlue = '#2e77fc';
-export const checkOrange = '#f2994a';
-export const checkError = '#fa555f';
-export const inProgressYellow = '#efac51';
-export const completedGreen = '#5cae73';
-export const separationGray = '#E5E5E5';
-export const brandSecondary = '#DFE4F4';
-export const backgroundMain = '#F7F8FD';
 // Check Design System Colors
-export const textPrimary = '#1F1F1F';
+export const brandMain = '#567bff';
+export const brandSecondary = '#3b5cd0';
+export const brandLight = '#cfdfff';
+export const brandAccent = '#293e86';// eslint-disable-line import/no-unused-modules
+export const brandBorder = '#d0d6ec';// eslint-disable-line import/no-unused-modules
+export const brandBackground = '#f1f5f6';
+export const brandHoverAccent = '#f2f8ff';// eslint-disable-line import/no-unused-modules
+
+export const textPrimary = '#1f1f1f';
 export const textSecondary = '#656565';
-export const brandMainCDS = '#567BFF';
-export const brandSecondaryCDS = '#3B5CD0';
-export const brandLightCDS = '#CFDFFF';
-export const alertLightCDS = '#FFF8ED';
-export const alertSecondaryCDS = '#A66300';
-export const alertMainCDS = '#E78A00';
-export const brandBackgroundCDS = '#F1F5F6';
-export const grayBorderCDS = '#CDD0D1';
-export const otherErrorMainCDS = '#F44336';
-export const validationLightCDS = '#F0FFF1';
-export const validationSecondaryCDS = '#237C27';
+export const textPlaceholder = '#b6b6b6';// eslint-disable-line import/no-unused-modules
+export const textDisabled = '#999';// eslint-disable-line import/no-unused-modules
+export const textLink = '#3b5cd0';// eslint-disable-line import/no-unused-modules
+
+export const validationMain = '#4caf50';
+export const validationSecondary = '#237c27';
+export const validationLight = '#f0fff1';
+
+export const alertMain = '#e78a00';
+export const alertSecondary = '#a66300';
+export const alertLight = '#fff8ed';
+
+export const errorMain = '#f44336';
+export const errorSecondary = '#c9291d';// eslint-disable-line import/no-unused-modules
+export const errorLight = '#ffeeed';// eslint-disable-line import/no-unused-modules
+
+export const grayBackground = '#f7f7f7';// eslint-disable-line import/no-unused-modules
+export const grayDisabledBackground = '#efefef';// eslint-disable-line import/no-unused-modules
+export const grayBorderMain = '#e4e4e4';
+export const grayBorderAccent = '#b4b4b4';// eslint-disable-line import/no-unused-modules
+
+export const otherWhite = '#fff';
```

Comment:
No difference at all @amoedoamorim
Reasons for this change:
- instead of using a variable name that collides with "standard" css generics like like `white` this can help find colors from the design system if we ever want to change them (not a great reason)
- when @pierreconti set up the color system in figma, this was an organizing term he selected. So I wanted a 1:1 relationship between colors in the code and colors in figma, so when a developer "inspects" a design in Figma they will see the same color name. For example, here it what you will see in figma: (slashes are just a figma organizing function)
<img width="290" alt="Screenshot 2023-02-24 at 9 59 12 AM" src="https://user-images.githubusercontent.com/418001/221210843-48893f51-e56c-4252-a170-556857b93bee.png">

So to answer your question...not much other than being comprehensive

Reply:
Thanks @brianfleming ! That makes sense! It's  an hierarchical thing (Other > White) rather than "this alternative white"

Reply:
Sheesh @amoedoamorim how did you do a much better job explaining that is much less text than I did?? 🙃